src/pool/srv_layout.h

```diff
@@ -83,6 +83,14 @@ struct pool_hdl {
 	uint64_t	ph_flags;
 	uint64_t	ph_sec_capas;
 	char		ph_machine[MAXHOSTNAMELEN+1];
+	size_t		ph_cred_len;
+	char		ph_cred[];
```

Comment:
might worth to add version inside pool_hdl, instead of rely on size.

Reply:
Don't you think it is enough to keep the version in pool service ?